src/components/PageNavHeader/pageNavHeader.test.tsx

```diff
@@ -0,0 +1,39 @@
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+import PageNavHeader from "./pageNavHeader";
+import { navigate } from "../../utils/test/mocks/navigate";
+
+const renderComponent = () => {
+  render(<PageNavHeader heading="name" backgroundColor="red" />);
+};
+
+describe("render pageNavHeader correctly", () => {
+  test("should display the header correctly", () => {
+    renderComponent();
+    const headingElement = screen.getByRole("heading");
+    const arrowElement = screen.getByTitle("left-caret");
+
+    expect(arrowElement).toBeInTheDocument();
+    expect(headingElement).toBeInTheDocument();
+  });
```

Comment:
Remove render test

Reply:
but this is among the test case  you add on trello